rules/terraform_locals_order.go

```diff
@@ -0,0 +1,124 @@
+package rules
+
+import (
+	"fmt"
+	"github.com/hashicorp/hcl/v2"
+	"sort"
+	"strings"
+
+	"github.com/hashicorp/hcl/v2/hclsyntax"
+	"github.com/hashicorp/hcl/v2/hclwrite"
+	"github.com/terraform-linters/tflint-plugin-sdk/tflint"
+	"github.com/terraform-linters/tflint-ruleset-terraform/project"
+)
+
+// TerraformLocalsOrderRule checks whether comments use the preferred syntax
+type TerraformLocalsOrderRule struct {
+	tflint.DefaultRule
+}
+
+// NewTerraformLocalsOrderRule returns a new rule
+func NewTerraformLocalsOrderRule() *TerraformLocalsOrderRule {
+	return &TerraformLocalsOrderRule{}
+}
+
+// Name returns the rule name
+func (r *TerraformLocalsOrderRule) Name() string {
+	return "terraform_locals_order"
+}
+
+// Enabled returns whether the rule is enabled by default
+func (r *TerraformLocalsOrderRule) Enabled() bool {
+	return false
```

Comment:
Instead of returning false here, it's recommended to set it to true and only add to the "all" preset.
This makes the rule available only to users who want all rules enabled.